docs/appActions.md

```diff
@@ -306,11 +306,11 @@ Change a hostPattern's config
 **temp**: `boolean`, Whether to change temporary or persistent
   settings. defaults to false (persistent).
 
-**skipSync**: `boolean`, Set true if a site isn't eligible for Sync (e.g. if this update was triggered by Sync)
+**skipSync**: `boolean`, Set true if a site isn't eligible for Sync (e.g. if addSite was triggered by Sync)
```

Comment:
i guess this is not correct, but it was auto-generated by the post-commit hook based on the jsdoc in js/actions/appActions.